pkgs/tools/misc/autorandr/default.nix

```diff
@@ -29,6 +29,8 @@ in
 
       make install TARGETS='autostart_config' PREFIX=$out DESTDIR=$out
 
+      make install TARGETS='manpage' PREFIX=$out
```

Comment:
Would it make any difference if we used installmManPage here? I am not sure.

Reply:
It's not necessary, because the upstream Makefile places it already to the correct location. If, for example, upstream adds another man page later, this `make install` will install it automatically. If we used `installManPage`, we would have to modify the derivation to install the new page.

Reply:
we may want to install man-pages to a dedicated `man` output though.

Reply:
I added the `man` output.